storey/redis_driver.py

```diff
@@ -59,7 +59,6 @@ class RedisDriver(NeedsRedisAccess, Driver):
 
     def __init__(self,
                  redis_client: Optional[Union[redis.Redis, redis.cluster.RedisCluster]] = None,
-                 redis_type: RedisType = RedisType.STANDALONE,
```

Comment:
i believe class RedisType can be removed altogether.

Reply:
ok